src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/TeamActionsMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import { DotsThreeVertical, Eye, Lock, Pencil } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { Link } from 'react-router-dom';
+
+import { Action } from '~constants/actions.ts';
+import { useActionSidebarContext } from '~context/ActionSidebarContext/ActionSidebarContext.ts';
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { tw } from '~utils/css/index.ts';
+import { formatText } from '~utils/intl.ts';
+import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts.ts';
+
+const displayName =
+  'v5.frame.ColonyHome.ReputationChart.partials.TeamActionsMenu';
+
+const MSG = defineMessages({
+  viewAllTeams: {
+    id: `${displayName}.viewAllTeams`,
+    defaultMessage: 'View all teams',
+  },
+  createTeam: {
+    id: `${displayName}.createTeam`,
+    defaultMessage: 'Create new team',
+  },
+  editTeam: {
+    id: `${displayName}.editTeam`,
+    defaultMessage: 'Edit team',
+  },
+});
+
+// @TODO maybe decouple some of this into a dropdown menu component?
+// eslint-disable-next-line max-len
+const dropdownItemClassName = tw`flex w-full items-center gap-2 rounded-s px-3 py-2 text-gray-900 hover:font-medium md:hover:bg-gray-50`;
```

Comment:
Here we should add also `text-md`

Current implementation
![Screenshot 2024-09-17 at 13 30 26](https://github.com/user-attachments/assets/e888f82d-4f6c-453f-9352-8cb5ddb164f1)

Figma
![Screenshot 2024-09-17 at 13 33 46](https://github.com/user-attachments/assets/08c91572-0c70-41b3-8b63-2a28d2d3772f)